Unix/configure

```diff
@@ -166,9 +166,10 @@ do
       enable_werror=1
       ;;
 
-    --enable-wchar)
-      enable_wchar=1
-      ;;
+## Disabling support for wchar builds
+##    --enable-wchar)
+##      enable_wchar=1
+##      ;;
```

Comment:
In general, I'm not a fan of this. Git is a revision control system, and it's trivial to revert a commit. Rather than leaving out commented sections here, removing 257 below, etc, I prefer just getting rid of it with an obvious commit message. Then we can find it and revert it if it was a mistake.

Leaving this here just makes people wonder what's going on. And you'd need to revert to get 257 back anyway. So rather than have historical commented out sections, keep it clean and revert later if needed.